sway-lib-core/src/num.sw

```diff
@@ -71,3 +71,21 @@ impl u8 {
         8
     }
 }
+
+impl b256 {
+     /// The smallest value that can be represented by this type.
+    pub fn min() -> b256 {
+        0x0000000000000000000000000000000000000000000000000000000000000000
+    }
+
+    /// The largest value that can be represented by this type,
+    /// 2<sup>8</sup> - 1.
```

Comment:
this isn't right

Reply:
weird! will fix, no idea where that came from.

Reply:
No, the `<sup>` was needed. See the Rust source code. Please revert. What's not right is that this line only here says `8` instead of `256`.